DECLARING_DEPENDENCIES.md

```diff
@@ -0,0 +1,210 @@
+# Declaring Dependencies on Google Java Libraries
+
+Google maintains a number of open source Java libraries that make it
+easier to use services in Google Cloud Platform (GCP). Additionally
+Google maintains several foundational libraries that can be used for
+very general purposes, which the GCP libraries also depend on. This
+document explains how to use bills of materials (BOMs) and other
+strategies to avoid dependency conflicts in these libraries.
+
+These recommendations apply to the following libraries:
+
+- [guava](https://github.com/google/guava)
+- [protobuf](https://github.com/protocolbuffers/protobuf)
+- [grpc-java](https://github.com/grpc/grpc-java)
+- [gax-java](https://github.com/googleapis/gax-java)
+- [google-http-java-client](https://github.com/googleapis/google-http-java-client)
+- [google-oauth-java-client](https://github.com/googleapis/google-oauth-java-client)
+- [google-api-java-client](https://github.com/googleapis/google-api-java-client)
+- [google-cloud-java](https://github.com/googleapis/google-cloud-java)
+- [beam](https://github.com/apache/beam)
+
+## Ensuring Compatibility
+
+To ensure that your own project uses compatible versions of these
+libraries, follow the guidance below for your build system.
+
+### Maven
+
+#### Use the requireUpperBoundDeps enforcer rule
+
+Dependency conflicts can happen with Maven even when libraries follow
+semver because Maven's dependency mediation algorithm can select older
+versions instead of newer versions, meaning new features will be
+missing that other libraries depend on. The `requireUpperBoundDeps`
+enforcer rule can be used to automatically discover incorrect version
```

Comment:
"can be used to automatically discover incorrect version selection  because it fails the build if anything other than the most
recent version is chosen" --> "can fail the build when an older version is chosen instead of a newer one."

Reply:
I think cutting the first part of the sentence makes the sentence as a whole disconnected from the prior one. I'll adjust the second half of the sentence, using your form as a model.